integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/steps/BridgeSteps.java

```diff
@@ -29,73 +31,83 @@ public BridgeSteps(TestContext context) {
         this.context = context;
     }
 
-    @Given("get list of Bridge instances fails with HTTP response code (\\d+)$")
-    public void authenticationIsEnabled(int responseCode) {
+    @Given("^the list of Bridge instances fails with HTTP response code (\\d+)$")
+    public void listOfBridgeInstancesReturnsHTTPResponseCode(int responseCode) {
         BridgeResource.getBridgeListResponse(context.getManagerToken())
                 .then()
                 .statusCode(responseCode);
     }
 
-    @When("create a Bridge with randomly generated name with access token")
-    public void createRandomBridge() {
-        String bridgeName = Utils.generateId("bridge");
+    @When("^a new Bridge \"([^\"]*)\" is created$")
+    public void newBridgeIsCreated(String testBridgeName) {
+        String systemBridgeName = Utils.generateId("bridge");
```

Comment:
Personally I would use `testBridgeName` as part of bridge id (maybe with `ci-` prefix), to be able to identify what bridge is related to what test in case of some fail, WDYT?

Reply:
yes, is a good idea :)

Reply:
done and added `test-` prefix